core/src/main/resources/db/changelog/changesets/2023_05_24_navbar_item.yaml

```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      id: navbar_item
+      author: nwatts
+      changes:
+        - renameColumn:
+            tableName: navbar_item
+            oldColumnName: label
+            newColumnName: text
+        - dropColumn:
+            tableName: navbar_item
+            columns:
+              - column:
+                  name: external_link_url
+              - column:
+                  name: anchor_link_path
```

Comment:
Nothing currently uses these columns, so we can drop and replace them instead of renaming / merging data.